engine/src/main/resources/templates/Advisor/deactivatedRecommendationInstantEmailBodyV2.html

```diff
@@ -0,0 +1,59 @@
+{@boolean renderSection1=true}
+{@boolean renderButtonSection1=true}
+{#include Common/insightsEmailBody}
+{#content-title}
+    Advisor - Red Hat Enterprise Linux
+{/content-title}
+{#content-title-section1}
+    Deactivated recommendations
+{/content-title-section1}
+{#content-title-right-part-section1}
+    <a href="{environment.url}/insights/advisor" target="_blank">{action.events.size()}</a>
+{/content-title-right-part-section1}
+{#content-button-section1}
+    <a target="_blank" href="{environment.url}/insights/advisor">Go to Insight Advisor</a>
+{/content-button-section1}
+{#content-body-section1}
+{#let recs=action.events.size()}
+<p>
+    {#if recs > 1}
+    These recommendations have recently been deactivated by Red Hat Insights and are no longer affecting your systems.
+    {#else}
+    This recommendation has recently been deactivated by Red Hat Insights and is no longer affecting your systems.
+    {/if}
+</p>
+
+<table class="rh-data-table rh-m-bordered">
+    <thead>
+    <tr>
+        <th>Recommendation{#if recs > 1}s{/if}</th>
+        <th>Total Risk</th>
+        <th>Systems</th>
+        <th>Reason</th>
+    </tr>
+    </thead>
+    <tbody>
+    {#each action.events}
+        <tr>
+            <td>{it.payload.rule_description}</td>
+            <td class="rh-severity">
+                {#switch it.payload.total_risk}
+                {#case 1}
+                <img src="https://console.redhat.com/apps/frontend-assets/email-assets/img_low.png" alt="Low severity" width="55" border="0">
+                {#case 2}
+                <img src="https://console.redhat.com/apps/frontend-assets/email-assets/img_moderate.png" alt="Moderate severity" width="85" border="0">
+                {#case 3}
+                <img src="https://console.redhat.com/apps/frontend-assets/email-assets/img_important.png" alt="Important severity" width="86" border="0">
+                {#case 4}
+                <img src="https://console.redhat.com/apps/frontend-assets/email-assets/img_critical.png" alt="Critical severity" width="70" border="0">
+                {/switch}
+            </td>
+            <td>{it.payload.affected_systems}</td>
+            <td>{it.payload.deactivation_reason}</td>
+        </tr>
+    {/each}
+    </tbody>
+</table>
+{/let}
+{/content-body-section1}
+{/include}
```

Comment:
Git is not happy because there's no empty line at the end of a few files in this PR 😄

Reply:
right, sorry :)